fontforge_font_creator/yaml_generator.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+import os
+
+excluded_ranges = list(range(0x0000, 0x0021)) + list(range(0x007F, 0x00A1))
+
+
+class Ranger:
+    def __init__(self, start, stop, excluded_range=None):
+        self.current = start
+        self.high = stop
+        self.stop = stop
+        self.excluded_range = excluded_range
+
+    def __iter__(self):
+        return self
+
+    def __next__(self):
+        if self.current > self.stop:
+            raise StopIteration
+        else:
+            self.current += 1
+            while self.current in self.excluded_range:
```

Comment:
Should we use [`unicodedata.category(chr(self.current))`](https://docs.python.org/3/library/unicodedata.html#unicodedata.category) instead of hard-coded exclusion ranges? Not sure which of [the categories](http://www.fileformat.info/info/unicode/category/index.htm) to include or exclude, though, but I guess

- `Co` Other, Private Use
- `LC` Letter, Cased
- `Ll` Letter, Lowercase
- `Lo` Letter, Other
- `Lt` Letter, Titlecase
- `Lu` Letter, Uppercase
- `Nd` Number, Decimal Digit
- `Sc` Symbol, Currency
- `Sm` Symbol, Math
- `So` Symbol, Other
- `Zs` Separator, Space

should be safe for our purposes.